administrator/components/com_config/model/form/application.xml

```diff
@@ -6,33 +6,33 @@
 		<field
 			name="caching"
 			type="list"
-			default="2"
 			label="COM_CONFIG_FIELD_CACHE_LABEL"
 			description="COM_CONFIG_FIELD_CACHE_DESC"
+			default="2"
 			filter="integer"
 			>
 			<option value="0">COM_CONFIG_FIELD_VALUE_CACHE_OFF</option>
-			<option value="1">COM_CONFIG_FIELD_VALUE_CACHE_CONSERVATIVE</option>
```

Comment:
Why did you change the ordering here? It looks good to me bevor the change ;) but maybe i miss something :)